templates/content/node--oe-tender--teaser.html.twig

```diff
@@ -0,0 +1,15 @@
+{#
+/**
+ * @file
+ * Theme override to display a node of bundle news in the teaser view mode.
```

Comment:
This needs to be rephrased, this is not about news content type.

Reply:
Updated